packages/expo-updates-interface/ios/EXUpdatesInterface.podspec

```diff
@@ -23,15 +23,7 @@ Pod::Spec.new do |s|
     'DEFINES_MODULE' => 'YES',
     'SWIFT_COMPILATION_MODE' => 'wholemodule'
   }
-
-  if ENV['USE_FRAMEWORKS']
-    compatibility_header_path = '${BUILT_PRODUCTS_DIR}/${PRODUCT_MODULE_NAME}.framework/Headers/${PRODUCT_MODULE_NAME}-Swift.h'
-  else
-    compatibility_header_path = '${BUILT_PRODUCTS_DIR}/Swift Compatibility Header/${PRODUCT_MODULE_NAME}-Swift.h'
-  end
-  s.script_phase = {
-    :name => 'Copy Swift Header',
-    :script => "COMPATIBILITY_HEADER_PATH=\"#{compatibility_header_path}\"" + '; ditto "${COMPATIBILITY_HEADER_PATH}" "${PODS_ROOT}/Headers/Public/${PRODUCT_MODULE_NAME}/${PRODUCT_MODULE_NAME}-Swift.h"',
-    :execution_position => :after_compile
```

Comment:
@wschurman sorry i have to revert your work for this, because it breaks use_frameworks! mode actually. hopefully the changes in this pr makes sense to you.